frontend/src/assets/main.scss

```diff
@@ -180,3 +180,8 @@ td .wrap-block {
     cursor: pointer;
   }
 }
+.p-input-icon-clear-right {
+  position: absolute !important;
+  right: 0rem;
+  line-height: 1.2;
```

Comment:
Probably want to make this fill the whole space instead of having that tiny gap below it. Removing line-height should do this. Could also do height: 100% as a safety net.

Reply:
`line-height` is removed. it seems to be inheriting height.